src/site/content/en/blog/introducing-model-viewer/index.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: 'Announcing <model-viewer> 1.0'
+subhead: 3D models for your web page as easily as writing HTML.
+authors:
+  - rkochman
+  - joemedley
+description: |
+3D models are more relevant than ever. Retailers bring in-store shopping
+experiences to customers' homes. Museums let anyone see artifacts from anywhere.
+3D is difficult to do without a deep knowledge of 3D technologies or third-party
+hosting. <model-viewer> 1.0 makes these tasks as easy as writing HTML.
+date: 2020-08-20
+hero: hero.jpg
+alt: A 3d image of a shark.
+tags:
+  - blog
+  - 3d
+  - model-viewer
+---
+
+3D models are more relevant than ever. Retailers bring in-store shopping
+experiences to customers' homes. Museums are making 3D models of their artifacts
+available to everyone on the web. Unfortunately, it can be difficult to add a 3D
+model to a website in a way that provides a great user experience without a deep
+knowledge of 3D technologies or resorting to hosting 3D content on a third-party
+site. The `<model-viewer>` web component, [introduced in early
+2019](../model-viewer), seeks to
+make putting 3D models on the web as easy as writing a few lines of HTML. Since
+then, the team has been working to address feedback and requests from the
+community. The culmination of that work is `<model-viewer>` version 1.0.
+
+## What's new since last year?
+
+Version 1.0 includes built-in support for augmented reality (AR) on the web,
+improvements to speed and fidelity, and other frequently-requested features.
+
+### Augmented reality
+
+Viewing a 3D model on a blank canvas is great, but being able to view it in your
+space is even better. Now `<model-viewer>` supports [WebXR augmented
+reality](https://modelviewer.dev/examples/webxr.html0), for an
+entirely-within-the-browser 3D and AR experience on Chrome Android.
+
+```html
+<model-viewer src="ToyTrain.glb" ar ar-scale="auto" camera-controls alt="A 3D model of a wooden toy train">
+</model-viewer>
+```
+<iframe style="width:100%; height: 100%;position: absolute; top: 50%; left: 50%; transform: translate(-50%,-50%);" src="https://modelviewer.dev/webdotdev/webxr.html" frameborder="0" allowfullscreen></iframe>
```

Comment:
Instead of that, should we show a video of what the user should see? Especially if we talk about AR, we may want everyone to see what it does, even laptop users. We could put a link for people to try. It would solve the current bug.

Reply:
I just sent you a video on chat. Let's swap the iframe to a link as Mounir suggested.

Reply:
@jpmedley the embedded example wouldn't work.